ts/components/services/ServicesSectionsList.tsx

```diff
@@ -35,7 +36,9 @@ type Props = AnimatedProps & OwnProps;
 
 const styles = StyleSheet.create({
   contentWrapper: {
-    flex: 1
+    flex: 1,
+    borderTopWidth: Platform.OS === "android" ? 0.19 : undefined,
```

Comment:
What causes this borderTopWidth value to differ in this component compared to MessagesInbox?